handlers.py

```diff
@@ -24,6 +24,19 @@ def handle(self, bot, update):
         raise NotImplementedError("Not implemented")
 
 
+class ListDefinitionsHandler(PersistenceAwareHandler):
+
+    def __init__(self, definition_repo):
+        super().__init__()
+        self.definitionRepo = definition_repo
+
+    def handle(self, bot, update):
+        definitions = self.definitionRepo.findAll()
+        reply = update.message.reply_text
+        for definition in definitions:
+            reply("/wtf {}".format(definition))
```

Comment:
formatted string should be in parentheses.

Reply:
Not sure what you mean, looks like it works as is